docs/destroy-cluster.md

```diff
@@ -0,0 +1,69 @@
+Only use this command if you are 100% sure that the cluster and its data should be destroyed.
+
+## `destroy-cluster` command
+
+To destroy a cluster, run the command:
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace rook-ceph? If absolutely certain, enter: yes-really-destroy-cluster
+```
+
+Any other response will cause the command to abort.
+
+## Example of destroying a cluster
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace "rook-ceph"? If absolutely certain, enter: yes-really-destroy-cluster
+yes-really-destroy-cluster
+Info: proceeding
+Info: Getting resource kind cephclusters
+Info: removing resource cephclusters: my-cluster
+Info: resource "my-cluster" is not yet deleted, applying patch to remove finalizer...
+Info: resource my-cluster was deleted
+Info: Getting resource kind cephblockpoolradosnamespaces
+Info: resource cephblockpoolradosnamespaces was not found on the cluster
+Info: Getting resource kind cephblockpools
+Info: removing resource cephblockpools: builtin-mgr
+Info: resource "builtin-mgr" is not yet deleted, applying patch to remove finalizer...
+Info: resource builtin-mgr was deleted
+Info: Getting resource kind cephbucketnotifications
+Info: resource cephbucketnotifications was not found on the cluster
+Info: Getting resource kind cephbuckettopics
+Info: resource cephbuckettopics was not found on the cluster
+Info: Getting resource kind cephclients
+Info: resource cephclients was not found on the cluster
+Info: Getting resource kind cephcosidrivers
+Info: resource cephcosidrivers was not found on the cluster
+Info: Getting resource kind cephfilesystemmirrors
+Info: resource cephfilesystemmirrors was not found on the cluster
+Info: Getting resource kind cephfilesystems
+Info: removing resource cephfilesystems: myfs
+Info: resource myfs was deleted
+Info: Getting resource kind cephfilesystemsubvolumegroup
+Info: the server could not find the requested resource: cephfilesystemsubvolumegroup
+Info: Getting resource kind cephnfses
+Info: resource cephnfses was not found on the cluster
+Info: Getting resource kind cephobjectrealms
+Info: resource cephobjectrealms was not found on the cluster
+Info: Getting resource kind cephobjectstores
+Info: removing resource cephobjectstores: my-store
+Info: resource my-store was deleted
+Info: Getting resource kind cephobjectstoreusers
+Info: resource cephobjectstoreusers was not found on the cluster
+Info: Getting resource kind cephobjectzonegroups
+Info: resource cephobjectzonegroups was not found on the cluster
+Info: Getting resource kind cephobjectzones
+Info: resource cephobjectzones was not found on the cluster
+Info: Getting resource kind cephrbdmirrors
+Info: resource cephrbdmirrors was not found on the cluster
+Info: removing deployment rook-ceph-tools
+Info: done
```

Comment:
The tool should wait to print `done` until the cluster cleanup jobs have completed on each node. Otherwise, the cleanup is not done yet. In my testing, I have realized the job took an extra minute or two to complete. The user really shouldn't try to create a new cluster until that cleanup job is completed, or else the cluster will not start properly.

Reply:
hey @travisn I added a timeout of two minutes in order to ensure that all the resources are deleted